pkg/didcomm/protocol/introduce/service.go

```diff
@@ -349,6 +351,10 @@ func (s *Service) newDIDCommActionMsg(md *metaData) service.DIDCommAction {
 			s.processCallback(md)
 		},
 		Stop: func(err error) { actionStop(customError{error: err}) },
+		Properties: &eventProps{
+			myDID:    md.MyDID,
```

Comment:
Why can't we just return connectionID instead of myDID and theirDID ? The consumers can find these details using the connectionID, if required. In most of the cases, the consumer would just be interested in knowing the sender. ie, `alice` or `bob` etc.

Reply:
myDID + theirDID are readily available in the protocol implementations (`HandleInbound(msg, myDID, theirDID)`)

Reply:
Yeah, but I feel those lower layer details are not meaningful to the consumers. As a consumer, I just want to know who sent me the message without knowing how to derive it by myself. I guess, it's a discussion about who(consumers or service) should fetch the connectionID from myDID and theirDID value.

Reply:
`connectionID` a lower level concept than DIDs.

Besides that, let's say we return the connectionID. The consumer would use `connection.Lookup.GetConnectionByID()` and get a `connection.Record`... where they see DIDs.

This solution is a good, speedy one that can be uniformly implemented with minimum hassle across all protocol services.

Reply:
> `connectionID` is a lower level concept than DIDs.
> 

I see this differently. DID is just a mean to connect to other agent. At higher level, the user would need the name/label of the sender. This label is tied to connectionID in the framework and connectionID in-turn has myDID and theirDID to help facilitate the communication.

> Besides that, let's say we return the connectionID. The consumer would use `connection.Lookup.GetConnectionByID()` and get a `connection.Record`... where they see DIDs.
> 
> This solution is a good, speedy one that can be uniformly implemented with minimum hassle across all protocol services.
I agree that myDID and theirDID is easy to implement.

Reply:
Both roads lead to `connection.Record`. It's easier with myDID + theirDID.